.travis.yml

```diff
@@ -16,6 +16,7 @@ rvm:
   - 2.2.3
   - 2.3.4
   - '2.4'
+  - 2.4.2
```

Comment:
It's not really worth testing 2.4.0 and 2.4.2. Would you mind getting rid of the `'2.4'` entry?